### PR TITLE
Feature start value

### PR DESCRIPTION
**Pull request by Wikunia**
Fix for #120

### CODE REVIEW

File: src/model.jl [hunk 1]
```diff
@@ -165,6 +166,8 @@ function create_root_model!(m::JuniperModel)
     ub = m.u_var
     # all continuous we solve relaxation first
     @variable(m.model, lb[i] <= x[i=1:m.num_var] <= ub[i])
+    println(m.start_value)
```
Comment (ccoffrin) [inline src/model.jl]: should this print statement be in here?

**Comment by ccoffrin** [comment]
Also, I am guessing that this is working the MOI wrapper becouse of the MINLPTests test this functionality, correct?

**Comment by Wikunia** [comment]
In MOI this is implemented here: https://github.com/lanl-ansi/Juniper.jl/blob/moi-wrapper/src/MOI_wrapper/variables.jl#L63-L75

File: src/model.jl (hunk 1 above)
Comment (Wikunia) [reply src/model.jl]: Deleted thanks

**Comment by ccoffrin** [comment]
LGTM, merge at you prefer.

**Pull request merged**